pkg/retry/retryer.go

```diff
@@ -274,7 +271,11 @@ func (rc *Container) initRetryer(method string, p Policy) error {
 	// NewRetryer can return nil if policy is nil
```

Comment:
This comment should be deleted, because after this change, NewRetryer will always return a non-nil Retryer if no error happens. So should be the `else` branch.